hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl

```diff
@@ -58,3 +58,29 @@ foreach alert_sender_inst [get_design_info -list instance -filter "prim_alert_se
   check_cov -waiver -add -instance "$alert_sender_inst"\
     -comment {FPV for Alerts has already been done elsewhere}
 }
+
+# Preconditions of OnehotCheck_A and Enable_A cannot happen without apply stopat.
+#
+# Below, we disable the assertions inside the embedded task and enable them inside the task where
+# the stopat on reg_we_cheeck lives.
+proc move_to_task {task_name assert_list precond_list} {
```

Comment:
I think this might be more difficult than you need. I've been using this:
```tcl
proc move_to_task {task_name assert_name} {
    task -edit ${task_name} -copy "${assert_name}*"
    assert -disable ${assert_name}
}
```

The asterisk means that I pick up all the preconditions for free, and disabling the assertion appears to disable its precondition cover as well. This might be quite a bit easier :-D

Reply:
Thanks, I agree :D